tools/flaky-test-reporter/config.go

```diff
@@ -34,10 +34,28 @@ const (
 )
 
 var (
-	jobConfigs = []JobConfig{
-		{"ci-knative-serving-continuous", "serving", prow.PostsubmitJob}, // CI flow for serving repo
+	// jobConfigs lists all repos and jobs to run within those repos
+	jobConfigs = map[string][]JobConfig{
+		// CI flows for serving repo
+		"serving": {{Name: "ci-knative-serving-continuous", Type: prow.PostsubmitJob, PostIssue: true},
+			{Name: "ci-knative-serving-istio-1.0.7-mesh", Type: prow.PostsubmitJob, PostIssue: true},
+			{Name: "ci-knative-serving-istio-1.0.7-no-mesh", Type: prow.PostsubmitJob, PostIssue: false},
+			{Name: "ci-knative-serving-istio-1.1.2-mesh", Type: prow.PostsubmitJob, PostIssue: false},
+			{Name: "ci-knative-serving-istio-1.1.2-no-mesh", Type: prow.PostsubmitJob, PostIssue: false},
+		},
 	}
-	repoIssueMap = map[string]string{
+	// slackChannelsMap lists which slack channel to post results in for each job in repo
+	slackChannelsMap = map[string]map[string][]slackChannel{
+		// channel mapping for serving repo
+		"serving": {"default": {{"api", "CA4DNJ9A4"}},
+			"ci-knative-serving-istio-1.0.7-mesh":    {{"networking", ""}},
+			"ci-knative-serving-istio-1.0.7-no-mesh": {{"networking", ""}},
+			"ci-knative-serving-istio-1.1.2-mesh":    {{"networking", ""}},
+			"ci-knative-serving-istio-1.1.2-no-mesh": {{"networking", ""}},
```

Comment:
I am thinking we should move this to a user defined config than hard-coding it here. So, we can update the config and not know about the code. WDYT?

Reply:
Good idea, will investigate.

Reply:
I think this is a big enough change to save for a future PR, if that's okay with you?